.gitignore

```diff
@@ -1,4 +1,5 @@
 helm/charts
 helm/Chart.lock
 ogdc-local-hostmount/
+OGDC_PV_HOST_PATH/
```

Comment:
The line above this one was supposed to cover the default PV host path, but I think that default was removed at some point.

Fine with leaving both of these here, but the `OGDC_PV_HOST_PATH` is supposed to be an envvar that gets set to something like `/users/user/ogdc-helm/ogdc-local-hostmount`.